packages/nc-gui-v2/nuxt.config.ts

```diff
@@ -50,7 +50,7 @@ export default defineNuxtConfig({
     css: {
       preprocessorOptions: {
         less: {
-          modifyVars: { 'primary-color': '#1348ba', 'text-color': 'rgba(61, 61, 61, 1)' },
+          modifyVars: { 'primary-color': '#4351e8', 'text-color': 'rgba(61, 61, 61, 1)' },
```

Comment:
You can also just import the color utils and use the actual primary color set in there
```ts
import { themeColors } from './utils/colorsUtils'

...
           modifyVars: { 'primary-color': themeColors.primary, 'text-color': 'rgba(61, 61, 61, 1)' },